src/services/user/services/AdminUsers.js

```diff
@@ -4,59 +4,18 @@ const { BadRequest, Forbidden } = require('@feathersjs/errors');
 const { authenticate } = require('@feathersjs/authentication').hooks;
 const logger = require('../../../logger');
 const { createMultiDocumentAggregation } = require('../../consent/utils/aggregations');
+const { modelServices: { prepareInternalParams } } = require('../../../utils');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['prepareInternalParams' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5680233)